lib/hyrax/configuration.rb

```diff
@@ -381,14 +381,20 @@ def google_analytics_id
 
     attr_writer :permission_levels
     def permission_levels
-      @permission_levels ||= { "Choose Access" => "none",
-                               "View/Download" => "read",
-                               "Edit" => "edit" }
+      @permission_levels ||= { "View/Download" => "read",
```

Comment:
Also... in order to satisfy @ggeisler 's text differences in the two cases (dropdowns to add a new person/group vs showing authority previously given to a user/group,) I needed to have the additional method. 

However there may be a cleaner way that I could use via i18n if I do the other changes for the other languages. I'll take a look at that in the next day or two as I have time between the other work.